src/extension.ts

```diff
@@ -4,9 +4,10 @@ import * as fs from 'fs';
 export function activate(context: vscode.ExtensionContext) {
 	let lastRealFilePath = '';
 
-	const disposable = vscode.window.onDidChangeActiveTextEditor(async ed => {
-		if (!ed || ed.document.uri.scheme !== 'file') return;
-		const filePath = ed.document.fileName!;
+	const disposable = vscode.window.onDidChangeTextEditorSelection(async (selection) => {
```

Comment:
Unfortunately switching to `onDidChangeTextEditorSelection` causes the symlink follow dialog to constantly popup when a selection occurs if a user clicks 'cancel'. This also causes this script run a lot more often than I'd like.

Maybe leaving it as `onDidChangeActiveTextEditor` and adding a temporary listener with `onDidChangeTextEditorSelection` to wait for the first emit would work.

Alternatively there might be a way to forward selection in `onDidChangeTextEditorSelection` or some other event through the URI.